src/Poseidon/Package.hs

```diff
@@ -576,6 +578,23 @@ newMinimalPackageTemplate baseDir name (GenotypeDataSpec format_ geno _ snp _ in
     ,   posPacDuplicate = 1
     }
 
+makePseudoPackageFromInGenotypeData :: InGenotypeData -> IO PoseidonPackage
+makePseudoPackageFromInGenotypeData (InGenotypeData format_ genoFile_ snpFile_ indFile_ snpSet_) = do
+    let genotypeData = GenotypeDataSpec format_ genoFile_ Nothing snpFile_ Nothing indFile_ Nothing (Just snpSet_)
+        baseDir = getBaseDir genoFile_ snpFile_ indFile_
+        pacName = takeBaseName genoFile_
```

Comment:
OK, I like this idea to just take the base name of the geno-file for the package name